bedrock/base/templates/base-error.html

```diff
@@ -11,9 +11,10 @@
 
 {% block site_header %}
   <header class="mzp-l-content header-spacing">
-    <div class="header-image">
-    {{ high_res_img('img/co-brand/firefox_moz_co-brand_sm.png', {'alt': 'Firefox, Mozilla'}) }}
+    <div class="header-image" style="width: 145px;">
+    <a href="{{ url('firefox') }}">{{ high_res_img('protocol/img/logos/firefox/logo-word-hor-sm.png', {'alt': 'Firefox, Mozilla','width':'155px'}) }}</a>
```

Comment:
Alt text should just be `Firefox` since this image doesn't combine the logos any more.